### ISSUE TITLE
Policy list reactions do not work (in an encrypted moderation room?)

### ISSUE THREAD

**Issue by heftig** [the-draupnir-project/Draupnir#563]
**Describe the bug**
The reactions presented when on the "Would you like to add the ban to a policy list?" message do nothing.

This is probably the same bug as https://github.com/matrix-org/mjolnir/issues/153.

**To Reproduce**
1. Ban a user.
3. Click on one of the question message's reactions in the management room representing a writable policy list.

**Expected behavior**
Draupnir reacts to the reaction. There's not even any log output.

**Additional context**
- Draupnir `v1.87.0` is set up through Pantalaimon `0.10.5` and the management room is encrypted.
- I've tried the buttons using Element Web and Element Android.
- All reaction events in the room, from the bot and the users, are unencrypted.

**Comment by Gnuxie** [comment]
Yeah this is an issue with Pantalaimon, i didn't know it already effected the abuse reporting in Mjolnir. That's interesting. 

https://the-draupnir-project.github.io/draupnir-documentation/bot/encryption

We don't support Pantalaimon anymore because of this issue. The alternative you have is tolerating it for now, upgrading to at least `2.0.0-beta.5` and enabling `experimentalRustCrypto` (which might not be much better). Or creating a new management room that is unencrypted. Sorry about this.
